File: java-advent-of-code/src/test/java/advent/twenty_fifteen/Day22WizardSimulator20xxTest.java
package advent.twenty_fifteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day22WizardSimulator20xxTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day22WizardSimulator20xx("/twenty_fifteen/Day22-WizardSimulator20xx-input.txt");

        assertEquals(900, dp.getPart1Answer().intValue());
        assertEquals(1216, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_fifteen/Day22WizardSimulator20xxTest.scala
package advent.twenty_fifteen

import org.scalatest.{FlatSpec, Matchers}

class Day22WizardSimulator20xxTest  extends FlatSpec with Matchers {
  "2015 Day 22 - Input" should "calculate answers" in {
    val dp = new Day22WizardSimulator20xx("/twenty_fifteen/Day22-WizardSimulator20xx-input.txt")
    dp.part1Answer should be(900)
    dp.part2Answer should be(1216)
  }
}
